packages/medusa/src/models/order.ts

```diff
@@ -540,6 +549,11 @@ export class Order extends BaseEntity {
  *     description: The total of gift cards with taxes
  *     type: integer
  *     example: 0
+ *  returnable_items:
+ *     description: The items that are returnable as part of the order, order swaps or order claims
```

Comment:
todo?: I believe we are missing:
```
Available if the relation `returnable_items ` is expanded.
```

Reply:
There is no relation for that, it is a computed fields like the other totals 👍